app/plugin_install.go

```diff
@@ -22,6 +22,23 @@ func (a *App) InstallPlugin(pluginFile io.ReadSeeker, replace bool) (*model.Mani
 }
 
 func (a *App) installPlugin(pluginFile io.ReadSeeker, replace bool) (*model.Manifest, *model.AppError) {
+	manifest, installErr := a.installPluginLocally(pluginFile, replace)
+	if installErr != nil {
+		return nil, installErr
+	}
+
+	// Store bundle in the file store to allow access from other servers.
+	pluginFile.Seek(0, 0)
+
+	storePluginFileName := filepath.Join("./plugins", manifest.Id) + ".tar.gz"
```

Comment:
suggest we build this as follows for clarity:
```
	storePluginFileName := filepath.Join("./plugins", fmt.Sprintf("%s.tar.gz", manifest.Id))
```

Reply:
Done